wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -316,7 +318,8 @@ public final class SessionManager: NSObject, SessionManagerType {
         pushTokenService: PushTokenServiceInterface = PushTokenService(),
         callKitManager: CallKitManagerInterface,
         isDeveloperModeEnabled: Bool = false,
-        isUnauthenticatedTransportSessionReady: Bool = false
+        isUnauthenticatedTransportSessionReady: Bool = false,
+        userDefaults: UserDefaults = .standard
```

Comment:
I thought we needed special UserDefaults that uses the shared group to share info between the two processes

Reply:
nevermind I saw where it was injected, maybe add a comment on the init or the prop so we don't forget it needs group shared UserDefaults

Reply:
I removed the default value and renamed the argument `sharedUserDefaults`. c8a59a522c5b2a06d5a9bf871a2a7e174175247d